src/main/java/com/aws/greengrass/util/orchestration/SystemServiceUtilsFactory.java

```diff
@@ -41,8 +43,41 @@ public synchronized SystemServiceUtils getInstance() {
                 return context.get(SystemdUtils.class);
             }
         } catch (IOException e) {
-            logger.atError().log("Unable to determine init process type");
+            logger.atDebug().log("No systemd is detected");
```

Comment:
do not change this, detect which init system to use properly

Reply:
The `procd` is a script instead of an executable file located in the file system. So it can't be detected in the same way as systemd. Besides, the solution has a bug. How can we tell which system management is used if there is a `systemd` in `/sbin/init` in some specific OpenWRT port? From my point of view, we should read pid 1 in a Linux-like platform to detect which system management is because that process is actually used.

Reply:
Will do